torch_frame/datasets/yandex.py

```diff
@@ -78,6 +78,7 @@ def get_df_and_col_to_stype(
             # if the numpy_dict contains only categorical or numerical features
             features = (categorical_features if categorical_features
                         is not None else numerical_features)
+            assert features is not None
```

Comment:
Raise an error with a good error message rather than using assert here?

Reply:
This error is for type narrowing, and I feel like this is more of an internal error that should never raised in user scripts.